conf.yaml

```diff
@@ -1118,9 +1118,9 @@ contents:
                         exclude_branches:   [ 0.7, 0.6]
                   - title:      APM .NET Agent
                     prefix:     dotnet
-                    current:    1.8
-                    branches:   [ master, 1.8 ]
-                    live:       [ master, 1.8 ]
+                    current:    1.9
+                    branches:   [ master, 1.9 ]
+                    live:       [ master, 1.9 ]
```

Comment:
I think we should keep the 1.8 branches listed in the documentation, even if they're flagged as superseded. Whilst we do encourage folks to move to the latest minor as soon as possible, being able to view docs specific to the major.minor is valuable.

Reply:
Ha, looks like I beat you by a minute to this! Agreed, and already fixed above :)